ballerina-shell/modules/shell-core/src/test/java/io/ballerina/shell/test/evaluator/PatternEvaluatorTest.java

```diff
@@ -36,7 +36,7 @@ public void testEvaluateTuple() throws BallerinaShellException {
         testEvaluate(TUPLE_EVALUATOR_TESTCASE);
     }
 
-    @Test
+//    @Test
```

Comment:
Commented due to https://github.com/ballerina-platform/ballerina-lang/issues/30570.

Reply:
@pubudu91 Please note.

Reply:
Ack. Will look in to this.